backend/tracim_backend/app_models/email_validators.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+from marshmallow import ValidationError
+from marshmallow.validate import Email
+
+
+class TracimEmailValidator(Email):
+    """ Special Email Validator who accept email from subdomain"""
+
+    USER_REGEX = re.compile(
+        r"(^[-!#$%&'*+/=?^`{}|~\w]+(\.[-!#$%&'*+/=?^`{}|~\w]+)*\Z"  # dot-atom
+        # quoted-string
+        r'|^"([\001-\010\013\014\016-\037!#-\[\]-\177]' r'|\\[\001-\011\013\014\016-\177])*"\Z)',
+        re.IGNORECASE | re.UNICODE,
+    )
+
+    DOMAIN_REGEX = re.compile(
+        # domain
+        r"(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.?)+" r"(?:[A-Z]{2,6}|[A-Z0-9-]{2,})\Z"
+        # literal form, ipv4 address (SMTP 4.1.3)
+        r"|^\[(25[0-5]|2[0-4]\d|[0-1]?\d?\d)" r"(\.(25[0-5]|2[0-4]\d|[0-1]?\d?\d)){3}\]\Z",
+        re.IGNORECASE | re.UNICODE,
+    )
```

Comment:
We need to be consistent between the frontend and the backend here. Please make sure the frontend part will run the same regular expression.
(cc @gbonaspetti)

Reply:
As I wrote in the issue, I think we should only check for `@`: email validation is **very** fragile.